### PR TITLE
QA-4048: launching RTA from oskar

### PR DESCRIPTION
**Pull request by dothebart**
This PR is related to this effort: https://github.com/arangodb/release-test-automation/pull/365

- [jenkins/runRTAtest.fish](https://github.com/arangodb/oskar/pull/523/files#diff-693f11ab86c5e45b3295282346b8fb6ef0fa89e888eef251ccbc6c6c757e35ef) is a copied & edited version of https://github.com/arangodb/oskar/blob/master/scripts/checkoutArangoDB.fish altered to do the very same for RTA as to do for the ArangoDB source.
- helper.fish patch wraps above new file
- https://github.com/arangodb/oskar/pull/523/files#diff-693f11ab86c5e45b3295282346b8fb6ef0fa89e888eef251ccbc6c6c757e35ef is the intended jenkins invokeable script. It is mostly similar to runPRtest2.fish, but adds downloading of 
  - the starter
  - the syncer
  - rclone
  - launch RTA against this.
- https://github.com/arangodb/release-test-automation/pull/365/files#diff-cfc685fd5d0867ed348db390084477cecae95997fb067bbe72f2f537502492ef is invoked in order to launch RTA by mounting

### CODE REVIEW

**Review by neunhoef — APPROVED** [review]
See one comment with a missing $WORKSPACE in an echo statement. Other than that: LGTM, I do not need to look again.

File: jenkins/runRTAtest.fish [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+and bash -x ./jenkins/oskar_tar.sh --edition $RTA_EDITION $argv
+
+set -l s $status
+
+# compiling results:
+moveResultsToWorkspace
+
+set -l matches $WORKDIR/work/release-test-automation/test_dir/*.{asc,testfailures.txt,deb,dmg,rpm,7z,tar.gz,tar.bz2,zip,html,csv,tar,png}
+for f in $matches
+   echo $f | grep -qv testreport ; and echo "mv $f" ; and mv $f $WORKSPACE; or echo "skipping $f"
```
Comment (neunhoef) [inline jenkins/runRTAtest.fish:33]: It seems that the echo part is missing $WORKSPACE, which would be confusing when inspecting logs.

**Review by neunhoef — APPROVED** [review]
Second try.

**Review by neunhoef — APPROVED** [review]
LGTM

**Review by naushniki — APPROVED** [review]
LGTM

**Review by fceller — APPROVED** [review]

**Pull request merged**